content/guides/core/automations/automation-events.md

```diff
@@ -60,9 +60,12 @@ From the project's **Automations** tab or directly from a line plot panel, you c
 - A [system metric]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}) such as `cpu`, which tracks the percentage of CPU utilization. W&B logs system metrics automatically every 15 seconds.
 
 The notification can watch the metric for these events:
+- **Run status change**: Trigger a workflow when a run changes its status to one or more of **Running**, **Failed**, or **Finished**.
```

Comment:
One tricky thing is that these run status change events aren't available when creating the automation in the workspace - it's only available from the automations tab (since it's not related to the specific metric chosen, it's related to the whole run)